### PR TITLE
Added 2 methods and a line of class doc

### PR DESCRIPTION
**Pull request by nrallakis**
Added addItem and removeItem methods and wrote a line on the List class documentation

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/scenes/scene2d/ui/List.java [hunk 1]
```diff
@@ -231,6 +234,28 @@ public void setItems (Array newItems) {
...
+		float oldPrefWidth = getPrefWidth(), oldPrefHeight = getPrefHeight();
+
+		items.add(newItem);
+		selection.validate();
+
+		invalidate();
+		if (oldPrefWidth != getPrefWidth() || oldPrefHeight != getPrefHeight()) invalidateHierarchy();
+	}
+	
+	public void removeItem (T... item) {
```
Comment (nooone) [inline gdx/src/com/badlogic/gdx/scenes/scene2d/ui/List.java]: I'd call it `removeItems` (plural) instead. Same with `addItems`

**Comment by nooone** [comment]
Both `items.add(newItem)` as well as `items.remove(item, false)` don't compile. Did you actually even try this yourself?

**Comment by deathbeam** [comment]
Probably change it to items.addAll and removeAll should compile?

**Comment by nrallakis** [comment]
I had tried it myself... i just forgot to remove the dots, thats why it doesn't compile my bad!

**Comment by nrallakis** [comment]
Now it compiles

**Pull request closed without merging**